conf/messages.en

```diff
@@ -125,7 +125,7 @@ howManyAsAContractPacker.checkYourAnswersLabel = HowManyAsAContractPacker
 howManyAsAContractPacker.error.lowBand.required = Enter the number of litres in the low band
 howManyAsAContractPacker.error.highBand.required = Enter the number of litres in the high band
 howManyAsAContractPacker.error.lowBand.nonNumeric = Number of litres in the low band must be numeric
-howManyAsAContractPacker.error.highBand.nonNumeric = Number of litres in the low band must be numeric
+howManyAsAContractPacker.error.highBand.nonNumeric = Number of litres in the high band must be numeric
 howManyAsAContractPacker.error.lowBand.negative = Number of litres must be 0 or more
 howManyAsAContractPacker.error.highBand.negative = Number of litres must be 0 or more
```

Comment:
Number of litres in the high band must be 0 or more. "in the high band " is missing